scripting/2.0/modules/ROOT/pages/scripting-module-xml-maven.adoc

```diff
@@ -70,12 +70,15 @@ Notice that since Scripting Module does not provide execution engines, you have
 For example, review the following list of engines provided by the previous version of Scripting Module:
 
 Groovy:
+
+[NOTE]
+Scripting module supports version 2.4.x for Groovy. Anypoint Studio recommends to use version 2.4.21.
```

Comment:
"Recommend" verb pattern is:
Recommend that...
Recommend doing sth...
Recommend  you to do...
In this case it should read:
Studio recommends using version...